src/wallet/init.cpp

```diff
@@ -15,6 +15,8 @@
 std::string GetWalletHelpString(bool showDebug)
 {
     std::string strUsage = HelpMessageGroup(_("Wallet options:"));
+    strUsage += HelpMessageOpt("-addresstype", _("What type of addresses to use (\"legacy\", \"p2sh\", \"bech32\", default: \"p2sh\")"));
+    strUsage += HelpMessageOpt("-changetype", _("What type of change outputs to use (\"legacy\", \"p2sh\", \"bech32\"), default: same as -addresstype"));
```

Comment:
Would it make sense to default `-changetype` to `bech32` if `-addresstype` is not specified?

Reply:
I think that's not acceptable for privacy reasons until bech32 is common on the network.

Reply:
Privacy is a good point. In that case, wouldn't it be better to use whichever address format is used for the recipient (unless the parameter is set)?

Reply:
`default: ` should be inside `()`.